aggregations/db_tables/daily_new_accounts_by_app_count.py

```diff
@@ -0,0 +1,80 @@
+import datetime
```

Comment:
If you need to add 2 separate statistics, it's better to create 2 separate reviews for that, it simplifies (read: makes faster) the review process

Reply:
Yeah, I totally get that for the future.  Sorry, this grew to 2 at the end of deployment.  The queries would run otherwise!

Neil needed something to test against.  Maybe this needs a dev server long-term?